host_vars/compute02.yml.example

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Rename to hv02 to align with readme / sample inventory file

Reply:
renamed.